workflows/iuf/stages.yaml

```diff
@@ -71,7 +71,7 @@ stages:
     type: product
     operations:
       - name: update-working-branch
```

Comment:
1. Is the name supposed to be `vcs-update-working-branch`? It is supposed to match the name in the template.
2. The local-path is actually not needed. We need to nuke it from stages.yaml. Just FYI.